service/controller/app/v1/resource/kubeconfigfinalizer/delete.go

```diff
@@ -0,0 +1,65 @@
+package kubeconfigfinalizer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if key.InCluster(cr) {
+		return nil
```

Comment:
Can we skip the logging in here I guess?

Reply:
Why should we?

Reply:
Because we already put the logs on `create.go` so I was thinking that I put too many logs on it, but you think it's fine?

Reply:
When we do some action we should properly log about it. Doesn't matter what happens during creation when this here is about deletion. Just put yourself into the shoes of somebody who debugs the operator and does not know how it works. So this somebody watches the logs and wants to understand what happens. When we do not log anything, nobody has a clue what the program is doing.

Reply:
Okay, I will put the same logs as creation, but adding `cancelling resource` as well.